src/ui/listview.rs

```diff
@@ -99,10 +101,30 @@ impl<I: ListItem> ListView<I> {
         }
     }
 
+    /// Return wether there are still items that aren't shown in the listview.
+    ///
+    /// `true` if there are unloaded items
+    /// `false` if all items are loaded
     fn can_paginate(&self) -> bool {
         self.get_pagination().max_content().unwrap_or(0) > self.get_pagination().loaded_content()
     }
 
+    /// Try to load more items into the list if neccessary.
+    #[inline]
+    fn try_paginate(&self) {
+        // Paginate if there are more items
+        //  AND
+        //   The selected item is the current last item (keyboard scrolling)
+        //    OR
+        //   The scroller can't scroll further down (mouse scrolling)
+        if self.can_paginate()
+            && (self.selected == self.content.read().unwrap().len().saturating_sub(1)
+                || !self.scroller.can_scroll_down())
```

Comment:
I think this will also trigger pagination if you use the keyboard to go down one line and the screen size is large enough to show all items, meaning it can't scroll down any further:
 
[Screencast from 09-15-2022 10:51:52 PM.webm](https://user-images.githubusercontent.com/19884/190506676-5d75597f-9a66-4877-91db-57583984fcbc.webm)

I can't think of a quick fix though, other than passing whether it was a mouse or keyboard move and preventing pagination in case it was a keyboard move, i.e. `!can_scroll_down() && mouse_event`.

But to be honest the inconvenience of handling this is worth it as it's almost not a problem, unless you have a nicer idea.

Reply:
Oh I thought it was supposed to load more items when that happened. Unless you mean the case where there are no more items to load, which is handled by the `can_paginate` method I think.

Reply:
It's not really a bug, just a change in behavior. But I think it's fine.